### PR TITLE
Added Mariana theme

### PR DESCRIPTION
**Pull request by bellamira**

### CODE REVIEW

File: VSColorThemes/Themes/Mariana.xml [hunk 1]
```diff
@@ -0,0 +1,8429 @@
...
+      <Color Name="LiveMetricsFailedRequestFill">
+        <Background Type="CT_RAW" Source="FF4C2728" />
+      </Color>
+      <Color Name="SquigglyOrange">
+        <Background Type="CT_RAW" Source="00FF9427" />
+      </Color>
+    </Category>
+    <Category Name="Autos" GUID="{a7ee6bee-d0aa-4b2f-ad9d-748276a725f6}">
+      <Color Name="ChangedText">
+        <Background Type="CT_INVALID" Source="00000000" />
```
Comment (Newrad0603) [inline VSColorThemes/Themes/Mariana.xml]: Please remove all instances of CT_INVALID color entries. They are non-functional, but bloat the private registry.

File: VSColorThemes/Themes/Mariana.xml [hunk 2]
```diff
@@ -0,0 +1,8429 @@
...
+        <Foreground Type="CT_RAW" Source="FFF7A1A1" />
+      </Color>
+      <Color Name="Plain Text">
```
Comment (Newrad0603) [inline VSColorThemes/Themes/Mariana.xml]: Once the CT_INVALID entries are removed, you'll end up with empty colors, so make sure to remove those as well. Once the empty colors are removed, you may have empty categories which should also be removed.

**Comment by Newrad0603** [comment]
Can you please provide an overview screenshot of the theme?

File: VSColorThemes/Themes/Mariana.xml (hunk 1 above)
Comment (bellamira) [reply VSColorThemes/Themes/Mariana.xml]: All set!

File: VSColorThemes/Themes/Mariana.xml (hunk 2 above)
Comment (bellamira) [reply VSColorThemes/Themes/Mariana.xml]: All set!

**Comment by Newrad0603** [comment]
@bellamira Sorry, I should have been clearer, I didn't mean you had to checkin the screenshots, I just wanted a comment with the screenshots in it. But other than that, the changes look good, so once the pngs are removed, we can close the PR.

**Comment by bellamira** [comment]
@Newrad0603 Oh, my bad. They have been removed and I have attached the image files to this comment.

MarianaCS
MarianaCSS
MarianaHTML
MarianaJS

**Pull request merged**